### PR TITLE
feature/bckhouse_xsec_flux

### PR DESCRIPTION
**Pull request by cjbackhouse**
Let's get this in before it gets lost forever. I would like to iterate on top of it.

### CODE REVIEW

**Comment by brucehoward-physics** [comment]
Agreed, actually I have some extra, semi-related things that could be foreseen to be a future PR building off this...

I've added reviewing this to my list of things 👍 Shouldn't be too hard since I've been using it or some version of it.

File: sbnana/CAFAna/test/test_xsec_flux.C [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const double N = m / (39.948*amu); // number of nuclei
+
+  std::cout << "V = " << V << ", m = " << m << " N = " << N << std::endl;
+
+  new TCanvas;
+  flux.ToTH1(1/N, kBlack, kSolid, kBinDensity)->Draw("hist");
+
+  //  new TCanvas;
+  TFile* fin = new TFile("/sbnd/app/users/bckhouse/dev/numi-at-icarus-flux-systematics/icarus_numi_flux_syst_ana.root");
+  if(fin->IsZombie()){
```
Comment (brucehoward-physics) [inline sbnana/CAFAna/test/test_xsec_flux.C:54]: I think this should be `!fin->IsZombie()` right?

**Review by brucehoward-physics — APPROVED** [review]
The changes here seem to basically match what I've been working with from Chris in one of my working areas, so I will mark this as accepted.

I've given a slight comment to be addressed on `test_xsec_flux.C` (I believe a slight bug) and I will note that I have used some different numbers here or there in my work, but as this is a test script I doubt it's worth worrying about being terribly precise.

File: sbnana/CAFAna/test/test_xsec_flux.C (hunk 1 above)
Comment (cjbackhouse) [reply sbnana/CAFAna/test/test_xsec_flux.C]: 🤦. Fixed

**Pull request merged**